app/views/layouts/application.html.erb

```diff
@@ -12,7 +12,7 @@
 <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
   <div class="container">
     <div class="navbar-header">
-      <a class="navbar-brand" href="#">KA Myanmar</a>
+      <a class="navbar-brand" href="/">KA Myanmar</a>
```

Comment:
Ditto for the hardcoded paths. Not really a problem with root, but we should keep consistent.